proto/info/info.proto

```diff
@@ -469,7 +469,8 @@ enum StorageStatus {
 message StorageDiskState {
   org.lfedge.eve.common.DiskDescription disk_name = 1;
   StorageStatus status = 2;            // In ZFS
-  string state = 3;                   // VDev aux state
+  string state = 3;                    // VDev aux state
+  string SmartStatus = 4;              // S.M.A.R.T. info availability
```

Comment:
Please use lower_snake_case.
Also check the anoted diffs in https://github.com/lf-edge/eve-api/pull/91/files which has all of the yetus issues.